v4/source/channels.yaml

```diff
@@ -803,7 +803,9 @@
       description: |
         Search public channels on a team based on the search term provided in the request body.
         ##### Permissions
-        Must have the `list_team_channels` permission.
+        If the user has the `list_team_channels` permission, the search will be performed within all the team's public channels.
+
+        Without the permission, the search will be performed within the channels that the user is a member of.
```

Comment:
Can you add a note about this only being the case in 5.16 and all previous versions you need the `list_team_channels` permission?

Reply:
Added!